relay-kafka/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "relay-kafka"
+authors = ["Sentry <oss@sentry.io>"]
+description = "Kafka related functionality for Relay"
+homepage = "https://getsentry.github.io/relay/"
+repository = "https://github.com/getsentry/relay"
+version = "22.9.0"
+edition = "2018"
+license-file = "../LICENSE"
+publish = false
+
+[dependencies]
+failure = "0.1.8"
+rdkafka = { version = "0.24", optional = true }
+rdkafka-sys = { version = "2.1.0", optional = true }
+relay-log = { path  = "../relay-log" }
+relay-statsd = { path  = "../relay-statsd", optional = true }
+rmp-serde = { version = "0.14.3", optional = true }
+serde = { version = "1.0.114", features = ["derive"] }
+serde_json = { version = "1.0.55", optional = true }
+
+[dev-dependencies]
+serde_yaml = "0.8.13"
+
+
+[features]
+default = []
+processing = [
```

Comment:
This feature effectively gives us producer implementations, should we name it as such?

Reply:
Yep, sounds good.